inst/shiny/tabs/outputs.R

```diff
@@ -161,7 +161,8 @@ output$meanplot <- renderPlotly({
                    selected_cycles = input$cyclesmean,
                    id_variable = input$selectidvar,
                    plot_ylog = input$logmeanplot,
-                   plot_sd = input$sdmeanplot) %>%
+                   plot_sd = input$sdmeanplot,
+                   plot_cv = input$cvmeanplot) %>%
```

Comment:
suggestion: I would suggest using `snake_case` for new names to be more readable. I know it will not be consistent with previous naming, but I think we should slowly implement better practices. If changing other related input names is not complicated (not used in many places), I would suggest changing them all within the scope. Also, if the name has multiple parts to it, please try going from the most general to the most specific:

```suggestion
                   plot_cv = input$mean_plot_cv) %>%
```